tests/ci_build/setup-test-machine.sh

```diff
@@ -9,53 +9,6 @@ die() {  echo "Error: ${*}" >&2;  exit 1; }
 
 [ ${UID} -eq 0 ] || die "Run this script as root"
 
-# Install necessary packages
-apt-get install fuse libfuse3-dev
-# necessary for debian
-apt-get install sudo valgrind git rsyslog psmisc fio
-apt-get install time bc
-apt-get install vim kmod
-
-# Install necessary packages for Ganesha tests
-apt-get install \
```

Comment:
Are these packages coming from somewhere else?

Reply:
Sorry, Github hid the changes in `setup_machine.sh`, so I didn't see the changes there.